clang/test/CodeGen/debug-info-system-typedef.c

```diff
@@ -0,0 +1,28 @@
+// Ensure that debug info for typedefs in system headers is still generated
+// even if -fno-system-debug is used.  This is justified because debug size 
+// savings is small, but debugging is commonly done with types that are
+// typedef-ed in system headers.  Thus, the increased debuggability
+// is worth the small extra cost.
+
+// Windows does not have <unistd.h>
+// UNSUPPORTED: system-windows
```

Comment:
Rather than disabling this test entirely on Windows, I suggest finding another typedef in a header file that is common to both Windows and Linux and using that typedef instead.

Reply:
Updated to check typedefs in stdint.h which is available on Windows.